doc/guidelines/development/jenkinsfile.rst

```diff
@@ -9,7 +9,7 @@ Jenkinsfile
 
 This uses the Jenkins shared library with the following parameters
 
-:name: This is the package name of the csc.
+:name: This is the package name of the csc (use an underscore or dash).
```

Comment:
Please suggest one or the other. Given your code that replaces _ with - while also referring to a github repo using the un-modified name, I think you had _ in mind all along?

Also please add an example (showing all arguments) for a non-trivial package.